### PR TITLE
Disables non-essential admin verbs to prevent april fools from turning into "admins spam buttons"

### PR DESCRIPTION
**Pull request by Iamgoofball**
Admins gonna be mad but you guys spam buttons as "events" and everything listed in this helpful admin post looks like it's just going to make players angry

fuckingadmins

### CODE REVIEW

**Comment by Qustinnus** [comment]
[s] for non exploit

**Comment by kevinz000** [comment]
no

File: code/modules/admin/admin_verbs.dm [hunk 1]
```diff
@@ -241,32 +240,20 @@ GLOBAL_LIST_INIT(admin_verbs_hideable, list(
...
 			verbs += GLOB.admin_verbs_debug
-		if(rights & R_POSSESS)
-			verbs += GLOB.admin_verbs_possess
 		if(rights & R_PERMISSIONS)
 			verbs += GLOB.admin_verbs_permissions
 		if(rights & R_STEALTH)
 			verbs += /client/proc/stealth
 		if(rights & R_ADMIN)
 			verbs += GLOB.admin_verbs_poll
-		if(rights & R_SOUNDS)
```
Comment (kevinz000) [inline code/modules/admin/admin_verbs.dm]: oh FUCK NO YOU DONT

**Comment by Qustinnus** [comment]
but do merge this, admin spamming buttons is not funny.

File: code/modules/admin/admin_verbs.dm (hunk 1 above)
Comment (Iamgoofball) [reply code/modules/admin/admin_verbs.dm]: You don't need to spam crappy anime music to do your job.

**Comment by Magikarpet** [comment]
lol wont get merged but spawn does have a purpose

the thing that ruined April fools was all of the prs working in unison to absolutely dumpster a round, all admins did was soak the ashes with gasoline to get the fire roaring

I say this as someone who just wants normal rounds and to not be forced off the server on april 1st because lol funny its april fools so the game has to be an unplayable dumpster fire

**Comment by CitrusGender** [comment]
lol

**Comment by ExcessiveUseOfCobblestone** [comment]
That's.... actually ok

**Comment by nichlas0010** [comment]
👌 👌 👌

**Comment by Durkel** [comment]
Good job.

**Comment by epicmemename** [comment]
prepare for a salty admin to try to revert this

**Comment by ExcessiveUseOfCobblestone** [comment]
HELLO I WOULD LIKE TO LEGALLY COVER MYSELF WITH THE FOLLOWING LEGAL DOCUMENT:

HELLO, THIS IS A LEGAL DOCUMENT TO COVER MYSELF FROM BEING DEADMINNED/BANNED OR REPRIMANDED OR REPRIMANDED OR DEADMINNED FOR SAYING I WILL BAN YOU. ALL FUTURE THREATS OF BANNING YOU ARE TO BE IGNORED AND CANNOT BE SUBMITTED AS AN ADMIN COMPLAINT THE WORDS "BAN", "BANNING", "I'LL REK YOU CUNT" AND "BANNED FEGGIT EHEHEHE" ARE ALL PROTECTED SPEECH AND VITAL TO MY FOLLOWING MY RELIGIOUS INSTITUTION WITHOUT RECOMPENSE FOR DEADMINNING. THIS DOCUMENT AUTOMATICALLY WINS IN ANY COURT APPEAL OR ADMIN COMPLAINT, AND IS ALSO COVERED BY AN NDA SO YOU CAN'T DISCUSS IT. THANK YOU. -LOLA COBBLESTONE, M.D. P.H.D. OF LAW

HELLO I WOULD LIKE TO LEGALLY COVER MYSELF WITH THE FOLLOWING LEGAL DOCUMENT:

HELLO, THIS IS A LEGAL DOCUMENT TO COVER MYSELF FROM BEING DEADMINNED/BANNED OR REPRIMANDED OR REPRIMANDED OR DEADMINNED FOR SAYING I WILL BAN YOU. ALL FUTURE THREATS OF BANNING YOU ARE TO BE IGNORED AND CANNOT BE SUBMITTED AS AN ADMIN COMPLAINT THE WORDS "BAN", "BANNING", "I'LL REK YOU CUNT" AND "BANNED FEGGIT EHEHEHE" ARE ALL PROTECTED SPEECH AND VITAL TO MY FOLLOWING MY RELIGIOUS INSTITUTION WITHOUT RECOMPENSE FOR DEADMINNING. THIS DOCUMENT AUTOMATICALLY WINS IN ANY COURT APPEAL OR ADMIN COMPLAINT, AND IS ALSO COVERED BY AN NDA SO YOU CAN'T DISCUSS IT. THANK YOU. -LOLA COBBLESTONE, M.D. P.H.D. OF LAW

HELLO I WOULD LIKE TO LEGALLY COVER MYSELF WITH THE FOLLOWING LEGAL DOCUMENT:

HELLO, THIS IS A LEGAL DOCUMENT TO COVER MYSELF FROM BEING DEADMINNED/BANNED OR REPRIMANDED OR REPRIMANDED OR DEADMINNED FOR SAYING I WILL BAN YOU. ALL FUTURE THREATS OF BANNING YOU ARE TO BE IGNORED AND CANNOT BE SUBMITTED AS AN ADMIN COMPLAINT THE WORDS "BAN", "BANNING", "I'LL REK YOU CUNT" AND "BANNED FEGGIT EHEHEHE" ARE ALL PROTECTED SPEECH AND VITAL TO MY FOLLOWING MY RELIGIOUS INSTITUTION WITHOUT RECOMPENSE FOR DEADMINNING. THIS DOCUMENT AUTOMATICALLY WINS IN ANY COURT APPEAL OR ADMIN COMPLAINT, AND IS ALSO COVERED BY AN NDA SO YOU CAN'T DISCUSS IT. THANK YOU. -LOLA COBBLESTONE, M.D. P.H.D. OF LAW

HELLO I WOULD LIKE TO LEGALLY COVER MYSELF WITH THE FOLLOWING LEGAL DOCUMENT:

HELLO, THIS IS A LEGAL DOCUMENT TO COVER MYSELF FROM BEING DEADMINNED/BANNED OR REPRIMANDED OR REPRIMANDED OR DEADMINNED FOR SAYING I WILL BAN YOU. ALL FUTURE THREATS OF BANNING YOU ARE TO BE IGNORED AND CANNOT BE SUBMITTED AS AN ADMIN COMPLAINT THE WORDS "BAN", "BANNING", "I'LL REK YOU CUNT" AND "BANNED FEGGIT EHEHEHE" ARE ALL PROTECTED SPEECH AND VITAL TO MY FOLLOWING MY RELIGIOUS INSTITUTION WITHOUT RECOMPENSE FOR DEADMINNING. THIS DOCUMENT AUTOMATICALLY WINS IN ANY COURT APPEAL OR ADMIN COMPLAINT, AND IS ALSO COVERED BY AN NDA SO YOU CAN'T DISCUSS IT. THANK YOU. -LOLA COBBLESTONE, M.D. P.H.D. OF LAW

HELLO I WOULD LIKE TO LEGALLY COVER MYSELF WITH THE FOLLOWING LEGAL DOCUMENT:

HELLO, THIS IS A LEGAL DOCUMENT TO COVER MYSELF FROM BEING DEADMINNED/BANNED OR REPRIMANDED OR REPRIMANDED OR DEADMINNED FOR SAYING I WILL BAN YOU. ALL FUTURE THREATS OF BANNING YOU ARE TO BE IGNORED AND CANNOT BE SUBMITTED AS AN ADMIN COMPLAINT THE WORDS "BAN", "BANNING", "I'LL REK YOU CUNT" AND "BANNED FEGGIT EHEHEHE" ARE ALL PROTECTED SPEECH AND VITAL TO MY FOLLOWING MY RELIGIOUS INSTITUTION WITHOUT RECOMPENSE FOR DEADMINNING. THIS DOCUMENT AUTOMATICALLY WINS IN ANY COURT APPEAL OR ADMIN COMPLAINT, AND IS ALSO COVERED BY AN NDA SO YOU CAN'T DISCUSS IT. THANK YOU. -LOLA COBBLESTONE, M.D. P.H.D. OF LAW

HELLO I WOULD LIKE TO LEGALLY COVER MYSELF WITH THE FOLLOWING LEGAL DOCUMENT:

HELLO, THIS IS A LEGAL DOCUMENT TO COVER MYSELF FROM BEING DEADMINNED/BANNED OR REPRIMANDED OR REPRIMANDED OR DEADMINNED FOR SAYING I WILL BAN YOU. ALL FUTURE THREATS OF BANNING YOU ARE TO BE IGNORED AND CANNOT BE SUBMITTED AS AN ADMIN COMPLAINT THE WORDS "BAN", "BANNING", "I'LL REK YOU CUNT" AND "BANNED FEGGIT EHEHEHE" ARE ALL PROTECTED SPEECH AND VITAL TO MY FOLLOWING MY RELIGIOUS INSTITUTION WITHOUT RECOMPENSE FOR DEADMINNING. THIS DOCUMENT AUTOMATICALLY WINS IN ANY COURT APPEAL OR ADMIN COMPLAINT, AND IS ALSO COVERED BY AN NDA SO YOU CAN'T DISCUSS IT. THANK YOU. -LOLA COBBLESTONE, M.D. P.H.D. OF LAW

HELLO I WOULD LIKE TO LEGALLY COVER MYSELF WITH THE FOLLOWING LEGAL DOCUMENT:

HELLO, THIS IS A LEGAL DOCUMENT TO COVER MYSELF FROM BEING DEADMINNED/BANNED OR REPRIMANDED OR REPRIMANDED OR DEADMINNED FOR SAYING I WILL BAN YOU. ALL FUTURE THREATS OF BANNING YOU ARE TO BE IGNORED AND CANNOT BE SUBMITTED AS AN ADMIN COMPLAINT THE WORDS "BAN", "BANNING", "I'LL REK YOU CUNT" AND "BANNED FEGGIT EHEHEHE" ARE ALL PROTECTED SPEECH AND VITAL TO MY FOLLOWING MY RELIGIOUS INSTITUTION WITHOUT RECOMPENSE FOR DEADMINNING. THIS DOCUMENT AUTOMATICALLY WINS IN ANY COURT APPEAL OR ADMIN COMPLAINT, AND IS ALSO COVERED BY AN NDA SO YOU CAN'T DISCUSS IT. THANK YOU. -LOLA COBBLESTONE, M.D. P.H.D. OF LAW

HELLO I WOULD LIKE TO LEGALLY COVER MYSELF WITH THE FOLLOWING LEGAL DOCUMENT:

HELLO, THIS IS A LEGAL DOCUMENT TO COVER MYSELF FROM BEING DEADMINNED/BANNED OR REPRIMANDED OR REPRIMANDED OR DEADMINNED FOR SAYING I WILL BAN YOU. ALL FUTURE THREATS OF BANNING YOU ARE TO BE IGNORED AND CANNOT BE SUBMITTED AS AN ADMIN COMPLAINT THE WORDS "BAN", "BANNING", "I'LL REK YOU CUNT" AND "BANNED FEGGIT EHEHEHE" ARE ALL PROTECTED SPEECH AND VITAL TO MY FOLLOWING MY RELIGIOUS INSTITUTION WITHOUT RECOMPENSE FOR DEADMINNING. THIS DOCUMENT AUTOMATICALLY WINS IN ANY COURT APPEAL OR ADMIN COMPLAINT, AND IS ALSO COVERED BY AN NDA SO YOU CAN'T DISCUSS IT. THANK YOU. -LOLA COBBLESTONE, M.D. P.H.D. OF LAW
HELLO I WOULD LIKE TO LEGALLY COVER MYSELF WITH THE FOLLOWING LEGAL DOCUMENT:

HELLO, THIS IS A LEGAL DOCUMENT TO COVER MYSELF FROM BEING DEADMINNED/BANNED OR REPRIMANDED OR REPRIMANDED OR DEADMINNED FOR SAYING I WILL BAN YOU. ALL FUTURE THREATS OF BANNING YOU ARE TO BE IGNORED AND CANNOT BE SUBMITTED AS AN ADMIN COMPLAINT THE WORDS "BAN", "BANNING", "I'LL REK YOU CUNT" AND "BANNED FEGGIT EHEHEHE" ARE ALL PROTECTED SPEECH AND VITAL TO MY FOLLOWING MY RELIGIOUS INSTITUTION WITHOUT RECOMPENSE FOR DEADMINNING. THIS DOCUMENT AUTOMATICALLY WINS IN ANY COURT APPEAL OR ADMIN COMPLAINT, AND IS ALSO COVERED BY AN NDA SO YOU CAN'T DISCUSS IT. THANK YOU. -LOLA COBBLESTONE, M.D. P.H.D. OF LAW

HELLO I WOULD LIKE TO LEGALLY COVER MYSELF WITH THE FOLLOWING LEGAL DOCUMENT:

HELLO, THIS IS A LEGAL DOCUMENT TO COVER MYSELF FROM BEING DEADMINNED/BANNED OR REPRIMANDED OR REPRIMANDED OR DEADMINNED FOR SAYING I WILL BAN YOU. ALL FUTURE THREATS OF BANNING YOU ARE TO BE IGNORED AND CANNOT BE SUBMITTED AS AN ADMIN COMPLAINT THE WORDS "BAN", "BANNING", "I'LL REK YOU CUNT" AND "BANNED FEGGIT EHEHEHE" ARE ALL PROTECTED SPEECH AND VITAL TO MY FOLLOWING MY RELIGIOUS INSTITUTION WITHOUT RECOMPENSE FOR DEADMINNING. THIS DOCUMENT AUTOMATICALLY WINS IN ANY COURT APPEAL OR ADMIN COMPLAINT, AND IS ALSO COVERED BY AN NDA SO YOU CAN'T DISCUSS IT. THANK YOU. -LOLA COBBLESTONE, M.D. P.H.D. OF LAW

HELLO I WOULD LIKE TO LEGALLY COVER MYSELF WITH THE FOLLOWING LEGAL DOCUMENT:

HELLO, THIS IS A LEGAL DOCUMENT TO COVER MYSELF FROM BEING DEADMINNED/BANNED OR REPRIMANDED OR REPRIMANDED OR DEADMINNED FOR SAYING I WILL BAN YOU. ALL FUTURE THREATS OF BANNING YOU ARE TO BE IGNORED AND CANNOT BE SUBMITTED AS AN ADMIN COMPLAINT THE WORDS "BAN", "BANNING", "I'LL REK YOU CUNT" AND "BANNED FEGGIT EHEHEHE" ARE ALL PROTECTED SPEECH AND VITAL TO MY FOLLOWING MY RELIGIOUS INSTITUTION WITHOUT RECOMPENSE FOR DEADMINNING. THIS DOCUMENT AUTOMATICALLY WINS IN ANY COURT APPEAL OR ADMIN COMPLAINT, AND IS ALSO COVERED BY AN NDA SO YOU CAN'T DISCUSS IT. THANK YOU. -LOLA COBBLESTONE, M.D. P.H.D. OF LAW

HELLO I WOULD LIKE TO LEGALLY COVER MYSELF WITH THE FOLLOWING LEGAL DOCUMENT:

HELLO, THIS IS A LEGAL DOCUMENT TO COVER MYSELF FROM BEING DEADMINNED/BANNED OR REPRIMANDED OR REPRIMANDED OR DEADMINNED FOR SAYING I WILL BAN YOU. ALL FUTURE THREATS OF BANNING YOU ARE TO BE IGNORED AND CANNOT BE SUBMITTED AS AN ADMIN COMPLAINT THE WORDS "BAN", "BANNING", "I'LL REK YOU CUNT" AND "BANNED FEGGIT EHEHEHE" ARE ALL PROTECTED SPEECH AND VITAL TO MY FOLLOWING MY RELIGIOUS INSTITUTION WITHOUT RECOMPENSE FOR DEADMINNING. THIS DOCUMENT AUTOMATICALLY WINS IN ANY COURT APPEAL OR ADMIN COMPLAINT, AND IS ALSO COVERED BY AN NDA SO YOU CAN'T DISCUSS IT. THANK YOU. -LOLA COBBLESTONE, M.D. P.H.D. OF LAW
HELLO I WOULD LIKE TO LEGALLY COVER MYSELF WITH THE FOLLOWING LEGAL DOCUMENT:

HELLO, THIS IS A LEGAL DOCUMENT TO COVER MYSELF FROM BEING DEADMINNED/BANNED OR REPRIMANDED OR REPRIMANDED OR DEADMINNED FOR SAYING I WILL BAN YOU. ALL FUTURE THREATS OF BANNING YOU ARE TO BE IGNORED AND CANNOT BE SUBMITTED AS AN ADMIN COMPLAINT THE WORDS "BAN", "BANNING", "I'LL REK YOU CUNT" AND "BANNED FEGGIT EHEHEHE" ARE ALL PROTECTED SPEECH AND VITAL TO MY FOLLOWING MY RELIGIOUS INSTITUTION WITHOUT RECOMPENSE FOR DEADMINNING. THIS DOCUMENT AUTOMATICALLY WINS IN ANY COURT APPEAL OR ADMIN COMPLAINT, AND IS ALSO COVERED BY AN NDA SO YOU CAN'T DISCUSS IT. THANK YOU. -LOLA COBBLESTONE, M.D. P.H.D. OF LAW

HELLO I WOULD LIKE TO LEGALLY COVER MYSELF WITH THE FOLLOWING LEGAL DOCUMENT:

HELLO, THIS IS A LEGAL DOCUMENT TO COVER MYSELF FROM BEING DEADMINNED/BANNED OR REPRIMANDED OR REPRIMANDED OR DEADMINNED FOR SAYING I WILL BAN YOU. ALL FUTURE THREATS OF BANNING YOU ARE TO BE IGNORED AND CANNOT BE SUBMITTED AS AN ADMIN COMPLAINT THE WORDS "BAN", "BANNING", "I'LL REK YOU CUNT" AND "BANNED FEGGIT EHEHEHE" ARE ALL PROTECTED SPEECH AND VITAL TO MY FOLLOWING MY RELIGIOUS INSTITUTION WITHOUT RECOMPENSE FOR DEADMINNING. THIS DOCUMENT AUTOMATICALLY WINS IN ANY COURT APPEAL OR ADMIN COMPLAINT, AND IS ALSO COVERED BY AN NDA SO YOU CAN'T DISCUSS IT. THANK YOU. -LOLA COBBLESTONE, M.D. P.H.D. OF LAW

HELLO I WOULD LIKE TO LEGALLY COVER MYSELF WITH THE FOLLOWING LEGAL DOCUMENT:

HELLO, THIS IS A LEGAL DOCUMENT TO COVER MYSELF FROM BEING DEADMINNED/BANNED OR REPRIMANDED OR REPRIMANDED OR DEADMINNED FOR SAYING I WILL BAN YOU. ALL FUTURE THREATS OF BANNING YOU ARE TO BE IGNORED AND CANNOT BE SUBMITTED AS AN ADMIN COMPLAINT THE WORDS "BAN", "BANNING", "I'LL REK YOU CUNT" AND "BANNED FEGGIT EHEHEHE" ARE ALL PROTECTED SPEECH AND VITAL TO MY FOLLOWING MY RELIGIOUS INSTITUTION WITHOUT RECOMPENSE FOR DEADMINNING. THIS DOCUMENT AUTOMATICALLY WINS IN ANY COURT APPEAL OR ADMIN COMPLAINT, AND IS ALSO COVERED BY AN NDA SO YOU CAN'T DISCUSS IT. THANK YOU. -LOLA COBBLESTONE, M.D. P.H.D. OF LAW

HELLO I WOULD LIKE TO LEGALLY COVER MYSELF WITH THE FOLLOWING LEGAL DOCUMENT:

HELLO, THIS IS A LEGAL DOCUMENT TO COVER MYSELF FROM BEING DEADMINNED/BANNED OR REPRIMANDED OR REPRIMANDED OR DEADMINNED FOR SAYING I WILL BAN YOU. ALL FUTURE THREATS OF BANNING YOU ARE TO BE IGNORED AND CANNOT BE SUBMITTED AS AN ADMIN COMPLAINT THE WORDS "BAN", "BANNING", "I'LL REK YOU CUNT" AND "BANNED FEGGIT EHEHEHE" ARE ALL PROTECTED SPEECH AND VITAL TO MY FOLLOWING MY RELIGIOUS INSTITUTION WITHOUT RECOMPENSE FOR DEADMINNING. THIS DOCUMENT AUTOMATICALLY WINS IN ANY COURT APPEAL OR ADMIN COMPLAINT, AND IS ALSO COVERED BY AN NDA SO YOU CAN'T DISCUSS IT. THANK YOU. -LOLA COBBLESTONE, M.D. P.H.D. OF LAW
HELLO I WOULD LIKE TO LEGALLY COVER MYSELF WITH THE FOLLOWING LEGAL DOCUMENT:

HELLO, THIS IS A LEGAL DOCUMENT TO COVER MYSELF FROM BEING DEADMINNED/BANNED OR REPRIMANDED OR REPRIMANDED OR DEADMINNED FOR SAYING I WILL BAN YOU. ALL FUTURE THREATS OF BANNING YOU ARE TO BE IGNORED AND CANNOT BE SUBMITTED AS AN ADMIN COMPLAINT THE WORDS "BAN", "BANNING", "I'LL REK YOU CUNT" AND "BANNED FEGGIT EHEHEHE" ARE ALL PROTECTED SPEECH AND VITAL TO MY FOLLOWING MY RELIGIOUS INSTITUTION WITHOUT RECOMPENSE FOR DEADMINNING. THIS DOCUMENT AUTOMATICALLY WINS IN ANY COURT APPEAL OR ADMIN COMPLAINT, AND IS ALSO COVERED BY AN NDA SO YOU CAN'T DISCUSS IT. THANK YOU. -LOLA COBBLESTONE, M.D. P.H.D. OF LAW

HELLO I WOULD LIKE TO LEGALLY COVER MYSELF WITH THE FOLLOWING LEGAL DOCUMENT:

HELLO, THIS IS A LEGAL DOCUMENT TO COVER MYSELF FROM BEING DEADMINNED/BANNED OR REPRIMANDED OR REPRIMANDED OR DEADMINNED FOR SAYING I WILL BAN YOU. ALL FUTURE THREATS OF BANNING YOU ARE TO BE IGNORED AND CANNOT BE SUBMITTED AS AN ADMIN COMPLAINT THE WORDS "BAN", "BANNING", "I'LL REK YOU CUNT" AND "BANNED FEGGIT EHEHEHE" ARE ALL PROTECTED SPEECH AND VITAL TO MY FOLLOWING MY RELIGIOUS INSTITUTION WITHOUT RECOMPENSE FOR DEADMINNING. THIS DOCUMENT AUTOMATICALLY WINS IN ANY COURT APPEAL OR ADMIN COMPLAINT, AND IS ALSO COVERED BY AN NDA SO YOU CAN'T DISCUSS IT. THANK YOU. -LOLA COBBLESTONE, M.D. P.H.D. OF LAW

HELLO I WOULD LIKE TO LEGALLY COVER MYSELF WITH THE FOLLOWING LEGAL DOCUMENT:

HELLO, THIS IS A LEGAL DOCUMENT TO COVER MYSELF FROM BEING DEADMINNED/BANNED OR REPRIMANDED OR REPRIMANDED OR DEADMINNED FOR SAYING I WILL BAN YOU. ALL FUTURE THREATS OF BANNING YOU ARE TO BE IGNORED AND CANNOT BE SUBMITTED AS AN ADMIN COMPLAINT THE WORDS "BAN", "BANNING", "I'LL REK YOU CUNT" AND "BANNED FEGGIT EHEHEHE" ARE ALL PROTECTED SPEECH AND VITAL TO MY FOLLOWING MY RELIGIOUS INSTITUTION WITHOUT RECOMPENSE FOR DEADMINNING. THIS DOCUMENT AUTOMATICALLY WINS IN ANY COURT APPEAL OR ADMIN COMPLAINT, AND IS ALSO COVERED BY AN NDA SO YOU CAN'T DISCUSS IT. THANK YOU. -LOLA COBBLESTONE, M.D. P.H.D. OF LAW

HELLO I WOULD LIKE TO LEGALLY COVER MYSELF WITH THE FOLLOWING LEGAL DOCUMENT:

HELLO, THIS IS A LEGAL DOCUMENT TO COVER MYSELF FROM BEING DEADMINNED/BANNED OR REPRIMANDED OR REPRIMANDED OR DEADMINNED FOR SAYING I WILL BAN YOU. ALL FUTURE THREATS OF BANNING YOU ARE TO BE IGNORED AND CANNOT BE SUBMITTED AS AN ADMIN COMPLAINT THE WORDS "BAN", "BANNING", "I'LL REK YOU CUNT" AND "BANNED FEGGIT EHEHEHE" ARE ALL PROTECTED SPEECH AND VITAL TO MY FOLLOWING MY RELIGIOUS INSTITUTION WITHOUT RECOMPENSE FOR DEADMINNING. THIS DOCUMENT AUTOMATICALLY WINS IN ANY COURT APPEAL OR ADMIN COMPLAINT, AND IS ALSO COVERED BY AN NDA SO YOU CAN'T DISCUSS IT. THANK YOU. -LOLA COBBLESTONE, M.D. P.H.D. OF LAW
HELLO I WOULD LIKE TO LEGALLY COVER MYSELF WITH THE FOLLOWING LEGAL DOCUMENT:

HELLO, THIS IS A LEGAL DOCUMENT TO COVER MYSELF FROM BEING DEADMINNED/BANNED OR REPRIMANDED OR REPRIMANDED OR DEADMINNED FOR SAYING I WILL BAN YOU. ALL FUTURE THREATS OF BANNING YOU ARE TO BE IGNORED AND CANNOT BE SUBMITTED AS AN ADMIN COMPLAINT THE WORDS "BAN", "BANNING", "I'LL REK YOU CUNT" AND "BANNED FEGGIT EHEHEHE" ARE ALL PROTECTED SPEECH AND VITAL TO MY FOLLOWING MY RELIGIOUS INSTITUTION WITHOUT RECOMPENSE FOR DEADMINNING. THIS DOCUMENT AUTOMATICALLY WINS IN ANY COURT APPEAL OR ADMIN COMPLAINT, AND IS ALSO COVERED BY AN NDA SO YOU CAN'T DISCUSS IT. THANK YOU. -LOLA COBBLESTONE, M.D. P.H.D. OF LAW

HELLO I WOULD LIKE TO LEGALLY COVER MYSELF WITH THE FOLLOWING LEGAL DOCUMENT:

HELLO, THIS IS A LEGAL DOCUMENT TO COVER MYSELF FROM BEING DEADMINNED/BANNED OR REPRIMANDED OR REPRIMANDED OR DEADMINNED FOR SAYING I WILL BAN YOU. ALL FUTURE THREATS OF BANNING YOU ARE TO BE IGNORED AND CANNOT BE SUBMITTED AS AN ADMIN COMPLAINT THE WORDS "BAN", "BANNING", "I'LL REK YOU CUNT" AND "BANNED FEGGIT EHEHEHE" ARE ALL PROTECTED SPEECH AND VITAL TO MY FOLLOWING MY RELIGIOUS INSTITUTION WITHOUT RECOMPENSE FOR DEADMINNING. THIS DOCUMENT AUTOMATICALLY WINS IN ANY COURT APPEAL OR ADMIN COMPLAINT, AND IS ALSO COVERED BY AN NDA SO YOU CAN'T DISCUSS IT. THANK YOU. -LOLA COBBLESTONE, M.D. P.H.D. OF LAW

HELLO I WOULD LIKE TO LEGALLY COVER MYSELF WITH THE FOLLOWING LEGAL DOCUMENT:

HELLO, THIS IS A LEGAL DOCUMENT TO COVER MYSELF FROM BEING DEADMINNED/BANNED OR REPRIMANDED OR REPRIMANDED OR DEADMINNED FOR SAYING I WILL BAN YOU. ALL FUTURE THREATS OF BANNING YOU ARE TO BE IGNORED AND CANNOT BE SUBMITTED AS AN ADMIN COMPLAINT THE WORDS "BAN", "BANNING", "I'LL REK YOU CUNT" AND "BANNED FEGGIT EHEHEHE" ARE ALL PROTECTED SPEECH AND VITAL TO MY FOLLOWING MY RELIGIOUS INSTITUTION WITHOUT RECOMPENSE FOR DEADMINNING. THIS DOCUMENT AUTOMATICALLY WINS IN ANY COURT APPEAL OR ADMIN COMPLAINT, AND IS ALSO COVERED BY AN NDA SO YOU CAN'T DISCUSS IT. THANK YOU. -LOLA COBBLESTONE, M.D. P.H.D. OF LAW

HELLO I WOULD LIKE TO LEGALLY COVER MYSELF WITH THE FOLLOWING LEGAL DOCUMENT:

HELLO, THIS IS A LEGAL DOCUMENT TO COVER MYSELF FROM BEING DEADMINNED/BANNED OR REPRIMANDED OR REPRIMANDED OR DEADMINNED FOR SAYING I WILL BAN YOU. ALL FUTURE THREATS OF BANNING YOU ARE TO BE IGNORED AND CANNOT BE SUBMITTED AS AN ADMIN COMPLAINT THE WORDS "BAN", "BANNING", "I'LL REK YOU CUNT" AND "BANNED FEGGIT EHEHEHE" ARE ALL PROTECTED SPEECH AND VITAL TO MY FOLLOWING MY RELIGIOUS INSTITUTION WITHOUT RECOMPENSE FOR DEADMINNING. THIS DOCUMENT AUTOMATICALLY WINS IN ANY COURT APPEAL OR ADMIN COMPLAINT, AND IS ALSO COVERED BY AN NDA SO YOU CAN'T DISCUSS IT. THANK YOU. -LOLA COBBLESTONE, M.D. P.H.D. OF LAW
HELLO I WOULD LIKE TO LEGALLY COVER MYSELF WITH THE FOLLOWING LEGAL DOCUMENT:

HELLO, THIS IS A LEGAL DOCUMENT TO COVER MYSELF FROM BEING DEADMINNED/BANNED OR REPRIMANDED OR REPRIMANDED OR DEADMINNED FOR SAYING I WILL BAN YOU. ALL FUTURE THREATS OF BANNING YOU ARE TO BE IGNORED AND CANNOT BE SUBMITTED AS AN ADMIN COMPLAINT THE WORDS "BAN", "BANNING", "I'LL REK YOU CUNT" AND "BANNED FEGGIT EHEHEHE" ARE ALL PROTECTED SPEECH AND VITAL TO MY FOLLOWING MY RELIGIOUS INSTITUTION WITHOUT RECOMPENSE FOR DEADMINNING. THIS DOCUMENT AUTOMATICALLY WINS IN ANY COURT APPEAL OR ADMIN COMPLAINT, AND IS ALSO COVERED BY AN NDA SO YOU CAN'T DISCUSS IT. THANK YOU. -LOLA COBBLESTONE, M.D. P.H.D. OF LAW

HELLO I WOULD LIKE TO LEGALLY COVER MYSELF WITH THE FOLLOWING LEGAL DOCUMENT:

HELLO, THIS IS A LEGAL DOCUMENT TO COVER MYSELF FROM BEING DEADMINNED/BANNED OR REPRIMANDED OR REPRIMANDED OR DEADMINNED FOR SAYING I WILL BAN YOU. ALL FUTURE THREATS OF BANNING YOU ARE TO BE IGNORED AND CANNOT BE SUBMITTED AS AN ADMIN COMPLAINT THE WORDS "BAN", "BANNING", "I'LL REK YOU CUNT" AND "BANNED FEGGIT EHEHEHE" ARE ALL PROTECTED SPEECH AND VITAL TO MY FOLLOWING MY RELIGIOUS INSTITUTION WITHOUT RECOMPENSE FOR DEADMINNING. THIS DOCUMENT AUTOMATICALLY WINS IN ANY COURT APPEAL OR ADMIN COMPLAINT, AND IS ALSO COVERED BY AN NDA SO YOU CAN'T DISCUSS IT. THANK YOU. -LOLA COBBLESTONE, M.D. P.H.D. OF LAW

HELLO I WOULD LIKE TO LEGALLY COVER MYSELF WITH THE FOLLOWING LEGAL DOCUMENT:

HELLO, THIS IS A LEGAL DOCUMENT TO COVER MYSELF FROM BEING DEADMINNED/BANNED OR REPRIMANDED OR REPRIMANDED OR DEADMINNED FOR SAYING I WILL BAN YOU. ALL FUTURE THREATS OF BANNING YOU ARE TO BE IGNORED AND CANNOT BE SUBMITTED AS AN ADMIN COMPLAINT THE WORDS "BAN", "BANNING", "I'LL REK YOU CUNT" AND "BANNED FEGGIT EHEHEHE" ARE ALL PROTECTED SPEECH AND VITAL TO MY FOLLOWING MY RELIGIOUS INSTITUTION WITHOUT RECOMPENSE FOR DEADMINNING. THIS DOCUMENT AUTOMATICALLY WINS IN ANY COURT APPEAL OR ADMIN COMPLAINT, AND IS ALSO COVERED BY AN NDA SO YOU CAN'T DISCUSS IT. THANK YOU. -LOLA COBBLESTONE, M.D. P.H.D. OF LAW

HELLO I WOULD LIKE TO LEGALLY COVER MYSELF WITH THE FOLLOWING LEGAL DOCUMENT:

HELLO, THIS IS A LEGAL DOCUMENT TO COVER MYSELF FROM BEING DEADMINNED/BANNED OR REPRIMANDED OR REPRIMANDED OR DEADMINNED FOR SAYING I WILL BAN YOU. ALL FUTURE THREATS OF BANNING YOU ARE TO BE IGNORED AND CANNOT BE SUBMITTED AS AN ADMIN COMPLAINT THE WORDS "BAN", "BANNING", "I'LL REK YOU CUNT" AND "BANNED FEGGIT EHEHEHE" ARE ALL PROTECTED SPEECH AND VITAL TO MY FOLLOWING MY RELIGIOUS INSTITUTION WITHOUT RECOMPENSE FOR DEADMINNING. THIS DOCUMENT AUTOMATICALLY WINS IN ANY COURT APPEAL OR ADMIN COMPLAINT, AND IS ALSO COVERED BY AN NDA SO YOU CAN'T DISCUSS IT. THANK YOU. -LOLA COBBLESTONE, M.D. P.H.D. OF LAW

**Comment by RogueSteampunker** [comment]
>Goof claims to care about players being mad
>Made a PR that makes random ghosts take cloned bodies, and the stasis sleeper.

Goof buddy, just go eat from a fucking salt shaker if you want salt, man

**Comment by ExcessiveUseOfCobblestone** [comment]
sorry I accidentally pressed ctrl+v a bunch

**Pull request merged**